README.md

```diff
@@ -36,6 +36,7 @@ libraryDependencies += "ch.epfl.scala" %% "collection-strawman" % "0.2.0-SNAPSHO
 - [x] `ArrayOps`
 - [x] `StringOps`
 - [x] `ArrayBuffer`
+- [x] `ImmutableArray` (new)
```

Comment:
I renamed `immutable.Array` to `immutable.ImmutableArray`. The redundancy seems less worrisome to me than reusing the name `Array`, which is in scope everywhere as `scala.Array`. WDYT?